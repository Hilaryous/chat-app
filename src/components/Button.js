// @flow
import styled from 'react-emotion'

export default styled('button')({
  padding: '12px',
  border: '4px #000000 solid',
  boxShadow: 'none',
  background: '#FBF6FE',
})
