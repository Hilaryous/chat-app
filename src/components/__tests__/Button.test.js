import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../Button'

describe('Button', () => {
  it('shows the correct snapshot tree for the Button component', () => {
    const tree = renderer.create(
      <Button>
          send
      </Button>,
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
