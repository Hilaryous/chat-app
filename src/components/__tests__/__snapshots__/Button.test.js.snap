// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button shows the correct snapshot tree for the Button component 1`] = `
<button
  className="css-bjolns"
>
  send
</button>
`;
